skimage/viewer/tests/test_widgets.py

```diff
@@ -88,6 +88,7 @@ def test_slider_float():
     assert_almost_equal(sld.val, 2.5, 2)
 
 
+@testing.skipif(True, reason="Don't know how to automate skipping.")

```

Comment:
Can you elaborate on this? LOL I don't understand why the previous skipif was insufficient?

Reply:
This is issue #3081 

The previous timer would close the QT window after 100 ms. But it doesn't seem to be working on my computer. Testing on travis is slow. I'll keep this on my todo after addressing your other comments.

Reply:
ping. ;) If you can't figure this out quickly, then write a better error message, something like "see issue #3081" ;)